docs/install/manual-install.md

```diff
@@ -166,6 +166,24 @@ manually).
 $ bin/update-all-reports
 {% endhighlight %}
 
+#### Junction lookup
```

Comment:
Thanks for adding this doc :) I wonder if this is the best place, it feels more like it goes under customising than installing to me, though not strongly (e.g. I imagine it won't apply to most people following the manual install doc, I mean). Perhaps under the "geocoder" part of customising?